components/tools/OmeroWeb/test/integration/test_thumbnails.py

```diff
@@ -23,6 +23,10 @@
 from builtins import range
 import base64
 import json
+from omero.model import RoiI, EllipseI, RectangleI, LineI, \
```

Comment:
minor: using alphabetical order will be better so it is easier to scan if there is a missing shape